manifests/adguard-v2/default-manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "AdGuard",
+  "version": "5.1.64",
+  "url": "https://static.adtidy.org/extensions/adguardadblocker/release/chrome.zip",
+  "sha512": "efe763ec7dc4144d7fd181c9bb824baef3af6882ae7cb51b58f039aba282a9920643acf26518fb2af7c6a21bd36edd42ccf7c1a374a01e9bd01fc3537a43f8cf",
```

Comment:
The final extension manifest can't have `url` and `sha512` fields. So we have to these fields during building that make less clear how the final manifest is look like.

WDYT about the following structure?
```
{ 
 "url": ..,
 "sha512": ..,
 "manifest-template" : {
   "name": ..,
   ..
 }
```
 ?

Reply:
It's actually not a manifest, just some config to download and check third-party extension. I think it would be better to completely rename these files

Reply:
Renamed to config.json